### PR TITLE
ember/ember-cli 3.18, ember-metrics 0.14, ember-cli-update, remove jQuery

### PR DESCRIPTION
**Pull request by rwwagner90**
This changes quite a few things. Runs the latest ember-cli-update, removes jQuery, updates various deps, runs latest blueprints etc.

### CODE REVIEW

**Comment by GreatWizard** [comment]
Thanks you :)

I update the CI and fix the issue with ember-auto-import in #15 
If y oucan do a tiny rebase and all will be fine 👍

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
```
Comment (GreatWizard) [inline .travis.yml:1]: We can delete the travis.yml file, we are not using travis

**Comment by rwwagner90** [comment]
@GreatWizard should be up to date now!

**Comment by GreatWizard** [comment]
Nice!

**Comment by rwwagner90** [comment]
@GreatWizard do you mind releasing a new version please?

**Comment by GreatWizard** [comment]
i'm waiting the documentation #16 but I will asap

**Pull request merged**